### PR TITLE
Testcontainers CircleCI fix

### PR DESCRIPTION
**Pull request by ao508**
Signed-off-by: Angelica Ochoa <15623749+ao508@users.noreply.github.com>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,11 +7,22 @@ version: 2.1
...
+    environment:
+      - TESTCONTAINERS_RYUK_DISABLED: true
     steps:
+      - setup_remote_docker:
+          version: 20.10.2
+      - run:
+          name: "Configure testcontainers.properties..."
+          command: |
+            touch /home/circleci/.testcontainers.properties && \
+            echo "DOCKER_HOST = unix:///var/run/docker.sock" > /home/circleci/.testcontainers.properties
```
Comment (ao508) [inline .circleci/config.yml:25]: just trying things out here. `testcontainers.properties`, if we actually need it, can be checked into `src/test/resources`

File: .circleci/config.yml [hunk 2]
```diff
@@ -7,11 +7,22 @@ version: 2.1
...
+
 jobs:
   run_checkstyle:
     <<: *defaults
     working_directory: /tmp/repos/cmo-metadb
+    environment:
+      - TESTCONTAINERS_RYUK_DISABLED: true
     steps:
+      - setup_remote_docker:
+          version: 20.10.2
```
Comment (bsideup) [inline .circleci/config.yml:20]: you don't need a remote Docker if you are using `machine: true`. In fact, that's most probably why it doesn't work :)
```suggestion
```

File: .circleci/config.yml (hunk 1 above)
Comment (bsideup) [inline .circleci/config.yml:25]: not needed. Testcontainers will use `unix:///var/run/docker.sock` automagically.
```suggestion
```

Comment (kiview) [reply .circleci/config.yml:25]: This should normally not be necessary and Testcontainers should detect the Docker environment by itself.

**Comment by bsideup** [comment]
Hey @ao508, I just left a couple of comments. I think the problem was with a mix of `machine: true` and an unnecessary `setup_remote_docker`. See Testcontainers' docs on how to use Testcontainers with CircleCI: https://www.testcontainers.org/supported_docker_environment/continuous_integration/circle_ci/

File: .circleci/config.yml [hunk 3]
```diff
@@ -7,11 +7,22 @@ version: 2.1
...
   maven: circleci/maven@1.0.3
 
+machine: true
+
 jobs:
   run_checkstyle:
     <<: *defaults
     working_directory: /tmp/repos/cmo-metadb
+    environment:
+      - TESTCONTAINERS_RYUK_DISABLED: true
```
Comment (bsideup) [inline .circleci/config.yml:17]: also unnecessary:
```suggestion
```

File: .circleci/config.yml (hunk 2 above)
Comment (ao508) [reply .circleci/config.yml:20]: thanks! yeah I've tried several variations with and without machine + with/without installing docker manually

Comment (ao508) [reply .circleci/config.yml:20]: i *do* think that the solution is getting the right docker installation going inside of the container, will update when I've played around with it a bit more

Comment (ao508) [reply .circleci/config.yml:20]: and D O N E 🙌

**Pull request merged**